src/map.cpp

```diff
@@ -1456,9 +1456,9 @@ void ServerMap::finishBlockMake(BlockMakeData *data,
 		if (!block)
 			continue;
 		/*
-			Update day/night difference cache of the MapBlocks
+			Update is air cache of the MapBlocks
 		*/
-		block->expireDayNightDiff();
+		block->expireIsAirCache();
 		/*
 			Set block as modified
 		*/
```

Comment:
`MOD_REASON_EXPIRE_DAYNIGHTDIFF` should be renamed too

(in practice it's pointless since all blocks that go through here should already be marked modified by the mapgen)